funsor/terms.py

```diff
@@ -440,12 +457,14 @@ def sample(self, sampled_vars, sample_inputs=None):
         If ``sample_inputs`` is provided, this creates a batch of samples
         scaled samples.
 
-        :param frozenset sampled_vars: A set of input variables to sample.
+        :param sampled_vars: A set of input variables to sample.
+        :type sampled_vars: str, Variable, or set or frozenset thereof.
```

Comment:
Does Sphinx render this `:type ...:` correctly?

Reply:
Yes, it sphinx does pretty well here:
![image](https://user-images.githubusercontent.com/648532/66966759-3a158d00-f033-11e9-8c21-84dfce123225.png)